resources/views/filament/components/server-console.blade.php

```diff
@@ -9,11 +9,41 @@
     <link rel="stylesheet" href="{{ asset('/css/filament/server/console.css') }}">
     @endassets
 
+    <div class="flex flex-auto gap-6" style="padding-bottom: 14px">
+        <div
+            class="grid grid-flow-row w-full p-3 rounded-lg shadow-sm overflow-hidden ring-1 ring-gray-950/5 dark:bg-gray-900 dark:ring-white/10">
+        <span>
+        <span class="text-md font-medium text-gray-500 dark:text-gray-400">
+            CPU:
+        </span>
+        <span class="text-md font-semibold">{{ $this->cpuUsage() }}</span>
+        </span>
+        </div>
+        <div
+            class="grid grid-flow-row w-full p-3 rounded-lg shadow-sm overflow-hidden ring-1 ring-gray-950/5 dark:bg-gray-900 dark:ring-white/10">
+        <span>
+            <span class="text-md font-medium text-gray-500 dark:text-gray-400">
+                Memory:
+            </span>
+            <span class="text-md font-semibold">{{ $this->memoryUsage() }}</span>
+        </span>
+        </div>
+        <div
+            class="grid grid-flow-row w-full p-3 rounded-lg shadow-sm overflow-hidden ring-1 ring-gray-950/5 dark:bg-gray-900 dark:ring-white/10">
+        <span>
+            <span class="text-md font-medium text-gray-500 dark:text-gray-400">
+                Disk:
+            </span>
+            <span class="text-md font-semibold">{{ $this->diskUsage() }}</span>
+        </span>
+        </div>
+    </div>
```

Comment:
Fix indentation.

Also, is there a reason these aren't widgets?

Reply:
Not really a reason, just kinda added it there.